src/renderer/windows/GPMWebView/interface/customUI.js

```diff
@@ -208,13 +216,32 @@ const fixChromecastButton = () => {
   cssRule('#player paper-icon-button[data-id="cast"] { display: inline-block; }');
 };
 
+let openSidebarStyles;
+const toggleOpenSidebar = (keepOpen) => {
```

Comment:
Purely because I'm sitting through a lecture right now talking about "semantics".

This should really be `setKeepSIdebarOpen` as it isn't technically toggling 👍